src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/KeyboardManagerViewModel.cs

```diff
@@ -1,241 +1,203 @@
-
-            return success;
-        }
-
-        private void FilterRemapKeysList(List<KeysDataModel> remapKeysList)
-        {
-            CombineRemappings(remapKeysList, (uint)VirtualKey.LeftControl, (uint)VirtualKey.RightControl, (uint)VirtualKey.Control);
-            CombineRemappings(remapKeysList, (uint)VirtualKey.LeftMenu, (uint)VirtualKey.RightMenu, (uint)VirtualKey.Menu);
-            CombineRemappings(remapKeysList, (uint)VirtualKey.LeftShift, (uint)VirtualKey.RightShift, (uint)VirtualKey.Shift);
-            CombineRemappings(remapKeysList, (uint)VirtualKey.LeftWindows, (uint)VirtualKey.RightWindows, Helper.VirtualKeyWindows);
-        }
-
-        private void CombineRemappings(List<KeysDataModel> remapKeysList, uint leftKey, uint rightKey, uint combinedKey)
-        {
-            KeysDataModel firstRemap = remapKeysList.Find(x => uint.Parse(x.OriginalKeys) == leftKey);
-            KeysDataModel secondRemap = remapKeysList.Find(x => uint.Parse(x.OriginalKeys) == rightKey);
-            if (firstRemap != null && secondRemap != null)
-            {
-                if (firstRemap.NewRemapKeys == secondRemap.NewRemapKeys)
-                {
-                    KeysDataModel combinedRemap = new KeysDataModel
-                    {
-                        OriginalKeys = combinedKey.ToString(),
-                        NewRemapKeys = firstRemap.NewRemapKeys,
-                    };
-                    remapKeysList.Insert(remapKeysList.IndexOf(firstRemap), combinedRemap);
-                    remapKeysList.Remove(firstRemap);
-                    remapKeysList.Remove(secondRemap);
-                }
-            }
-        }
-    }
-}
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.PowerToys.Settings.UI.Lib.Helpers;
+using Microsoft.PowerToys.Settings.UI.Lib.Utilities;
+using Microsoft.PowerToys.Settings.UI.Lib.ViewModels.Commands;
+
+namespace Microsoft.PowerToys.Settings.UI.Lib.ViewModels
+{
+    public class KeyboardManagerViewModel : Observable
+    {
+        private const string PowerToyName = "Keyboard Manager";
+        private const string RemapKeyboardActionName = "RemapKeyboard";
+        private const string RemapKeyboardActionValue = "Open Remap Keyboard Window";
+        private const string EditShortcutActionName = "EditShortcut";
+        private const string EditShortcutActionValue = "Open Edit Shortcut Window";
+        private const string JsonFileType = ".json";
+        private const string ProfileFileMutexName = "PowerToys.KeyboardManager.ConfigMutex";
+        private const int ProfileFileMutexWaitTimeoutMilliseconds = 1000;
+
+        private readonly FileSystemWatcher watcher;
```

Comment:
This variable isn't used in this file so it can be removed since the FileWatcher logic seems to be in the xaml.cs file.